crates/orchestrator/src/config.rs

```diff
@@ -585,7 +579,7 @@ pub struct HotShotConfigFile<KEY: SignatureKey> {
     pub known_nodes_with_stake: Vec<PeerConfig<KEY>>,
     #[serde(skip)]
     /// The known DA nodes' public key and stake values
-    pub known_da_nodes: HashSet<PeerConfig<KEY>>,
+    pub known_da_nodes: Vec<PeerConfig<KEY>>,
```

Comment:
Curious: why this change?

Reply:
Yeah so,

I spent a bit debugging why at one point these changes broke the DA validation. It turns out that the way we've implemented the QC, everyone has to have this in the same order. The HashSet was not preserving order when sent over the wire, so it was breaking validation

Reply:
Ahh, makes sense!